applications/search/src/pages/api-details-page/singleColGrid/singleColGrid.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
dette ser ut som et generell komponent, kan det ligge i components?

Reply:
moved to components